app/views/publishers/notifications/index.html.slim

```diff
@@ -1,22 +1,27 @@
 - content_for :page_title_prefix, t(".title", count: notifications.unread.total_count)
 
+- if notifications.any?
+  - content_for :skip_links do
+    a.govuk-skip-link href="#notifications-results" Skip to notifications
```

Comment:
I feel like we might want to put these in as translations, but I'm not hell bent on it.

Reply:
i think they should